packages/eth2-types/test/consistency.test.ts

```diff
@@ -1,6 +1,7 @@
+import {} from "mocha";
```

Comment:
woot? you can add describe, it to enable typing

Reply:
could just do `import "mocha"`